specifyweb/frontend/js_src/lib/components/QueryComboBox/index.tsx

```diff
@@ -518,6 +522,15 @@ export function QueryComboBox({
                 }
               />
             )}
+            {hasViewButton && (
```

Comment:
there is already a `<DataEntry.View>` on line 415 - could you combine this button with it by updating the condition?

Reply:
sorry if I wasn't clear, but you misunderstood what I meant.
I meant to remove the `<DataEntry.View>` on line 525 because there already is a `<DataEntry.View>` on line 415.
instead change condition on line 412+make necessary refactors so that you only have `<DataEntry.View>` in the code once, but it is displayed in all the cases when necessary (and `<DataEntry.Edit>` is still displayed when necessary. this also avoids the possible bug with `<DataEntry.View>` being displayed twice

ANOTHER solution that avoids duplication is to create a variable that like `const viewButton = <DataEntry.View ... />` and then use it in two places - in either case, code duplication is avoided.